Libraries/ActionSheetIOS/ActionSheetIOS.js

```diff
@@ -25,6 +36,13 @@ var ActionSheetIOS = {
       typeof callback === 'function',
       'Must provide a valid callback'
     );
+
+    var processedTintColor = this.processTintColor(options.tintColor);
+
+    if (processedTintColor) {
+      options.tintColor = processedTintColor;
+    }
```

Comment:
Yea this was made based on @ide comment here. 
https://github.com/facebook/react-native/pull/4590#discussion_r46765407